controllers/auth.go

```diff
@@ -332,6 +332,18 @@ func (c *ApiController) Login() {
 			}
 		}
 
+		var application *object.Application
+		application, err = object.GetApplication(fmt.Sprintf("admin/%s", authForm.Application))
```

Comment:
Is it safe to move this code out of the "else" branch of the if: `if authForm.Password == ""`? Will `authForm.Application` always be non-empty for all if branches?

Is this only a code refactoring, right? It doesn't seem to have any functional difference.